virtkvmqe/migration_test_plan/migration/test_scenarios/rhel7_10044.py

```diff
@@ -58,7 +58,9 @@ def run_case(params):
     test.main_step_log('4. Do live migration.')
     check_info = utils_migration.do_migration(remote_qmp=src_remote_qmp,
                                               migrate_port=incoming_port,
-                                              dst_ip=dst_host_ip)
+                                              dst_ip=dst_host_ip,
+                                              downtime_val=downtime,
+                                              speed_val=speed)
```

Comment:
NameError: global name 'speed' is not defined,
 there is no assignment for 'speed', so, I think we should use default value 32M/s as following:
speed_val='33554432', with this update, case pass

Reply:
Thanks, updated by assigning value for the variant in the beginning.